### DISCUSSION TITLE
(Un-related to Firefox UI Fix) I need your help Mr. black7375

### DISCUSSION THREAD

**Discussion by Aleksandra0056** [black7375/Firefox-UI-Fix#707] [General]
Hello, I need your help with [28135](https://code.videolan.org/videolan/vlc/-/issues/28135) on VLC issue tracker, because one of the key files in this VLC issue is a “.skm” file.

I'm asking this to you and not to anyone else, because SKM is South Korean Telecom 3GP Video File (.skm), and only you can help me to speed up solving VLC issue [28135](https://code.videolan.org/videolan/vlc/-/issues/28135) with attaching cloud download link which could contain a “.skm” file.

Well, you can help me and Mr. Jean Baptiste Kempf by uploading a “.skm” file into some cloud (That cloud should have English lang. by default), and add a cloud download link with the “.skm” file to the comments. 

If you don't have a mobile phone which could make a “.skm” file, then close this discussion as outdated, and if you have a mobile phone which could make a “.skm” file, then please upload a sample “.skm” video file to cloud and post link to file here, and I will send that file to VLC team.

It absolutely doesn't matter at all what the attached “.skm” video should contain, just upload a “.skm” video please.
(NOTE: The uploaded “.skm” file in cloud SHOULDN'T contain copyrighted material!)

Also, if someone else has a “.skm” file, then please attach the file link here, but I'm in this discussion asking only Mr. black7375, so everyone else, don't reply unless you have a “.skm” file link.

I will thank for any response, sir. And I'm sorry, that I'm posting off-topic discussions, but you are the only one which could help me, thank you.

**Comment by black7375** [comment]
As far as I know, it's an old format, so not many people use it and not many devices support `.skm`.
[file_example_MP4_1280_10MG.zip](https://github.com/black7375/Firefox-UI-Fix/files/11584505/file_example_MP4_1280_10MG.zip)

How to make and open this file?
- Sample file: https://file-examples.com/index.php/sample-video-files/sample-mp4-files/
  - 10MB, 1280x720
- Sample encoder: https://www.cacaotools.com/cacaoencoder/
- Sample player: https://www.gomlab.com/gomplayer-media-player/?language=eng

**Reply by Aleksandra0056** [reply]
Thank you very much, I love someone which is helping me with my work.

**Reply by Aleksandra0056** [reply]
My VLC 4.0 opened that file correctly. And I'm closing this discussion as solved.

**Reply by black7375** [reply]
![image](https://github.com/black7375/Firefox-UI-Fix/assets/25581533/218c310c-cd50-4aef-8639-f704aa398856)

I don't know exactly about the format of the video, but it seems to be able to `.k3g` files can also be converted into an cacao encoder.
Anyway, I hope my answer helps.

**Reply by Aleksandra0056** [reply]
You have .k3g file too? Can you also attach it here, please? Because we need .k3g too in mentioned VLC issue.

**Reply by Aleksandra0056** [reply]
No, I will download cacao encoder and encode it myself, I no longer need attaching .k3g, sorry for confusion.
